redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/actions/StartDatabaseServerAction.java

```diff
@@ -9,22 +9,22 @@
 import com.sequenceiq.cloudbreak.common.event.Selectable;
 import com.sequenceiq.redbeams.api.model.common.DetailedDBStackStatus;
 import com.sequenceiq.redbeams.flow.redbeams.common.RedbeamsContext;
-import com.sequenceiq.redbeams.flow.redbeams.common.RedbeamsEvent;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertReplaceInRedBeamsSuccess;
 import com.sequenceiq.redbeams.flow.redbeams.start.event.StartDatabaseServerRequest;
 import com.sequenceiq.redbeams.service.stack.DBStackStatusUpdater;
 
 @Component("START_DATABASE_SERVER_STATE")
-public class StartDatabaseServerAction extends AbstractRedbeamsStartAction<RedbeamsEvent> {
+public class StartDatabaseServerAction extends AbstractRedbeamsStartAction<CertReplaceInRedBeamsSuccess> {
```

Comment:
Please revert this class as per above; let us keep `START_DATABASE_SERVER_STATE` as the init state.